framework/src/mesh/MooseMesh.C

```diff
@@ -500,21 +500,23 @@ MooseMesh::cacheChangedLists()
 }
 
 ConstElemPointerRange *
-MooseMesh::refinedElementRange()
+MooseMesh::refinedElementRange() const
```

Comment:
Perhaps not but it the return type really shouldn't be modified! There's still more cleanup to make it completely const correct